spec/command/plugins_helper_spec.rb

```diff
@@ -6,6 +6,8 @@ module CLAide
   describe Command::PluginsHelper do
     extend SpecHelper::PluginsStubs
 
+    before { CLAide::Plugins.config = default_testing_config }
+
```

Comment:
Since another spec rewrites `CLAide::Plugins.config`, all specs in this file would sometimes fail depends on the order of specs.

https://github.com/dbgrandi/claide-plugins/blob/26f67ca2faf4c579777d9d42134f87172614a6d1/spec/command/plugins_spec.rb#L37-L41